packages/symgen/lib/src/executable.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Executable script to generate symbol mapping for shared libraries.
+
+import 'dart:io';
+import 'package:args/args.dart';
+
+import 'config.dart';
+import 'generator.dart';
+
+void main(List<String> args) {
+  final argResult = _parseArguments(args);
+
+  final configFile = File(argResult['config']);
+  if (!configFile.existsSync()) {
+    print('Error: $configFile not found.');
+    exit(1);
+  }
+
+  Config config;
+  try {
+    config = Config.fromYaml(configFile.readAsStringSync());
+  } catch (e) {
+    print(e);
+    print('Error: Please fix configuration errors and re-run the tool.');
+    exit(1);
+  }
+
+  final generator = CodeGenerator(config);
+  generator.generate();
+}
+
+ArgResults _parseArguments(List<String> args) {
+  final parser = ArgParser(allowTrailingOptions: true)
+    ..addFlag(
+      'help',
+      abbr: 'h',
+      help: 'Print this usage',
+      negatable: false,
+    )
+    ..addOption(
+      'config',
+      help: 'Path to Yaml file containing configurations',
+      mandatory: true,
+    );
```

Comment:
How about adding an option that takes a path to the allowlist file, or adding an entry
```yaml
allowlist: ${TIZEN40_WEARABLE_ALLOWLIST_PATH}
```
to `configs/4.0/wearable/symgen.yaml` file so that generating the bindings can be easily automated?

Reply:
👍 good idea.

Reply:
@swift-kim I added options to symgen to override `output`, `allowlist`, `library-path`.

Reply:
Thanks!